etl_extension/org_units/create_org_code_uid_table.sql

```diff
@@ -1,4 +1,5 @@
 CREATE TABLE IF NOT EXISTS org_code_uid (
   uid VARCHAR(32) PRIMARY KEY,
-  code TEXT DEFAULT NULL
+  code TEXT DEFAULT NULL,
+  path VARCHAR(255) DEFAULT NULL
```

Comment:
Please, wrap path in backquotes because a 'path' is a reserved word in SQL.

Reply:
Corrected.